setup.py

```diff
@@ -360,11 +363,11 @@ def finalize_options(self):
         pass
 
     def run(self):
-        with tempfile.NamedTemporaryFile(prefix='org.musicbrainz.Picard.appdata.',
+        with tempfile.NamedTemporaryFile(prefix='%s.appdata.' % PICARD_APP_ID,
                                          suffix='.xml') as tmp_file:
             self.spawn([
                 'msgfmt', '--xml',
-                '--template=org.musicbrainz.Picard.appdata.xml.in',
+                '--template=%s.in' % APPDATA_FILE,
```

Comment:
Since we have it here and below, it might make sense to have:

```python
PACKAGE_NAME = "picard"
APPDATA_FILE = PICARD_APP_ID + '.appdata.xml'
APPDATA_FILE_TEMPLATE = APPDATA_FILE + '.in'
```

And use:

```python
'--template=%s' % APPDATA_FILE_TEMPLATE,
```
And below:

```python
            '--language=appdata',
            APPDATA_FILE_TEMPLATE,
```